chart/openfaas/templates/controller-rbac.yaml

```diff
@@ -1,5 +1,5 @@
 {{- $functionNs := default .Release.Namespace .Values.functionNamespace }}
-{{- if eq .Values.operator.create false }}
+{{- if eq .Values.faasnetes.create true }}
```

Comment:
Hey there!

What about using {{- if .Values.faaslambda.create }}? It's easier to read and more consistent with the other parts of the chart :>